packages/sui-i18n/test/i18nSpec.js

```diff
@@ -156,6 +156,21 @@ describe('I18N', () => {
         expect(i18n.t(`${key}.literalOne`)).to.eql('TranslateDynamicEsES')
       })
 
+      describe('properly number format with extended options', () => {
+        beforeEach(() => {
+          i18n.culture = 'es-ES'
+          i18n.extendedIntlNumberOptions = {useGrouping: true}
+        })
+        afterEach(() => {
+          i18n.culture = ''
+          i18n.extendedIntlNumberOptions = null
+        })
+
+        it('should display grouping separators even if the locale prefers otherwise', () => {
+          expect(i18n.n(1000)).to.eql('1.000')
+        })
+      })
```

Comment:
since you agreed on this https://github.com/SUI-Components/sui/pull/1590/files#r1122916373 

you could add another test to make sure options specified as args on `.n` do override the _generic_ ones 😉